pkg/registry/tld_registry.go

```diff
@@ -0,0 +1,1458 @@
+package registry
+
+var PrivateTLDs = []string{
+	"google",
+}
+
+var TLDs = []string{
```

Comment:
What do you think if here we use embed for easier maintenance?
```go
import (
	_ "embed"
	"fmt"
	"strings"
)

//go:embed tld.txt
var tldData string

var tlds[]string

func init() {
	tldData = strings.Split(tldData , "\n")
	for i, tld:= range tlds{
		tlds[i] = strings.TrimSpace(tld)
	}
}
```

Reply:
yess!